### PR TITLE
Merge remote-tracking branch 'refs/remotes/jchannon/master'

### PR DESCRIPTION
**Pull request by pdwetz**
Initial xml docs for ValidationExtensions and QueryStringExtensions as part of issue #44

### CODE REVIEW

**Comment by jchannon** [comment]
Not sure what's gone on here. There are no changes. Wonder if it's a PR for you merging my branch to yours and back again somehow

**Comment by pdwetz** [comment]
Closing since I was an idiot and didn't include my commit.

**Comment by jchannon** [comment]
It's ok I can squash and merge

File: src/ModelBinding/ValidationExtensions.cs [hunk 1]
```diff
@@ -6,6 +6,11 @@ namespace Botwin.ModelBinding
 
     public static class ValidationExtensions
     {
+        /// <summary>
+        /// Retrieve formatted validation errors
+        /// </summary>
+        /// <param name="result"><see cref="ValidationResult"/></param>
+        /// <returns>Property names and associated error messages <see cref="IEnumerable{dynamic}"/></returns>
```
Comment (jchannon) [inline src/ModelBinding/ValidationExtensions.cs]: Maybe `<see cref="IEnumerable{dynamic}"/> of property names and associated error messages`

File: src/Request/QueryStringExtensions.cs [hunk 2]
```diff
@@ -7,6 +7,14 @@
 
     public static class QueryStringExtensions
     {
+        /// <summary>
+        /// Retrieve strongly typed query paramater value for given key
```
Comment (jchannon) [inline src/Request/QueryStringExtensions.cs]: Should be `parameter`

File: src/Request/QueryStringExtensions.cs [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
+        /// </summary>
+        /// <typeparam name="T">Query param type</typeparam>
+        /// <param name="query"><see cref="IQueryCollection"/></param>
+        /// <param name="key">Query param key</param>
+        /// <param name="defaultValue">Default quary paramter value</param>
```
Comment (jchannon) [inline src/Request/QueryStringExtensions.cs]: `Default value if key not found`

File: src/Request/QueryStringExtensions.cs [hunk 4]
```diff
@@ -19,6 +27,13 @@ public static T As<T>(this IQueryCollection query, string key, T defaultValue =
             return value.ConvertTo<T>();
         }
 
+        /// <summary>
+        /// Retrieve strongly typed query paramater values for given key
```
Comment (jchannon) [inline src/Request/QueryStringExtensions.cs]: Typo `Parameter`

**Comment by jchannon** [comment]
Thanks @pdwetz Great stuff! 👍

**Pull request merged**